dev/breeze/src/airflow_breeze/breeze.py

```diff
@@ -234,7 +235,7 @@
             {
                 "name": "Setup autocomplete flags",
                 "options": [
-                    "--force-setup",
+                    "--force§",
```

Comment:
Accidental?

Reply:
I unified it force as `pipx install uses --force as well - but yeah the § was accidental :)